src/connectors/status/status-db.connector.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Updates and stores status variables
+ * @module connectors/status
+ */
+const mongodb = require("mongodb");
+const { statusCollectionDouble } = require("./status-db.double");
+const { STATUSDB_URL } = require("../../config");
+const { logEmitter } = require("../../services/logging.service");
+
+let client;
+let statusDB;
+let statusCollection;
+
+/**
+ * Sets up a connection to the status collection in the config database.
+ * The client, configDB and statusCollection variables are accessible to other functions in this connector.
+ */
+const establishConnectionToMongo = async () => {
+  if (process.env.DOUBLE_MODE === "true") {
+    logEmitter.emit(
+      "doubleMode",
+      "status-db.connector",
+      "establishConnectionToMongo"
+    );
+    statusCollection = statusCollectionDouble;
+  } else {
+    client = await mongodb.MongoClient.connect(STATUSDB_URL, {
+      useNewUrlParser: true
+    });
+
+    statusDB = client.db("register_a_food_business_status");
+
+    statusCollection = statusDB.collection("status");
```

Comment:
The collection name should be "backEndStatus"

Reply:
The original plan was that is should be a common status collection and as such the the front end status and other future status were also going to be stored in this same collection